src/client/components/componentWithSliders/modalsavewindow/modalSaveWindow.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modalSaveWindow.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+
+const cancelWindowIcon = <FontAwesomeIcon icon={faTimes}/>;
+
+const SavePressetesModalWindow = (props) => (
+        <div className="modalSaveWindow">
+            <div className="headerModalWindow">
+                {cancelWindowIcon} 
+            </div>
+            <div className="textArea">
+                <strong>Please type presset`&#39;`s name</strong>             
```

Comment:
You don't need backticks here. You can write this element like this: "<strong>Please type presset&#39;s name</strong>"